songpal/main.py

```diff
@@ -252,6 +265,36 @@ def print_settings(settings, depth=0):
                 click.echo("    - %s" % out)
 
 
+@cli.command()
+@click.argument("zone", required=False)
+@click.argument("activate", required=False)
+@pass_dev
+@coro
+async def zone(dev: Device, zone, activate):
+    """Get and change outputs."""
+    zones = await dev.get_zones()
+    if zone:
+        try:
+            zone = next((x for x in zones if x.title == zone))
+            
+            power = 'inactive' if activate == "off" else 'active'
```

Comment:
Please modify `activate` to take a boolean as input, and define the click.argument as such. There should be no need for such constructs, not even in the cli parts :-)

Reply:
Okay, did that, although I can't say that `songpal zone 'Main Zone' true` feels especially natural compared to `songpal zone 'Main Zone' on`. Feels especially inconsistent in the context of `songpal power on`

Reply:
Ah, it indeed does not accept on or off. The proper fix would be to introduce a new type by extending the https://github.com/pallets/click/blob/master/click/types.py#L344 to check first if on or off is handled. This type should also be used directly in `power` command, but that can be left for another PR :-)

Reply:
Done!